src/main/java/seedu/duke/Duke.java

```diff
@@ -25,12 +27,18 @@ public static void main(String[] args) {
 
         new Duke();
         boolean isOngoing = true;
+        try {
+            freqFile.readFile();
+        } catch (CustomException e) {
+            Ui.showError(e);
+        }
```

Comment:
Is it possible to use an assert here instead of try-catch since the catch block is not doing anything? I'm not very sure

Reply:
reafFile could throw a filenotfoundexception and that exception has to be dealt with. 
We could refactor this into a method in the main function to make it look neater